

<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name="video_in" pkg="opencv3" type="image_saver" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw/compressed"/>
    <param name="autosize" value="true" />
  </node>
</launch>
